.github/workflows/main-build.yml

```diff
@@ -339,7 +337,7 @@ jobs:
         with:
           path: |
             ~/.m2/repository/io/opentelemetry/
-          key: ${{ runner.os }}-maven-local-${{ hashFiles('.github/patches/**/opentelemetry-java-*.patch') }}
+          key: ${{ runner.os }}-maven-local-${{ hashFiles('.github/patches/opentelemetry-java-*.patch') }}
```

Comment:
Is this key correct? This will not hash `.github/patches/opentelemetry-java.patch`

Reply:
Yes, thanks for highlighting this. I have now updated to `.github/patches/opentelemetry-java*.patch`